src/swap/defi/lifi.ts

```diff
@@ -405,6 +415,43 @@ export function makeLifiPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
 
         break
       }
+      case 'sui': {
+        // SUI uses pre-built transactions
+        const { data } = asTransactionRequestSui(transactionRequestRaw)
+
+        spendInfo = {
```

Comment:
Create a makeTxParams instead of spendInfo so we can utilize otherMethods.makeTx to handle the unsigned data blob